src/src/pages/Params/Params.tsx

```diff
@@ -116,21 +117,54 @@ const Params = ({
   onRowsVisibilityChange,
   onParamsScaleTypeChange,
 }: IParamsProps): React.FunctionComponentElement<React.ReactNode> => {
-  let scaleStates = selectedParams.reduce((acc, param) => {
-    (acc as any)[param.key] = param.scale;
-    return acc;
-  }, {});
+  let scaleStates = getDefaultScaleStates(highPlotData);
+  let newScaleStates = scaleStates;
+
+  function updateParamsState() {
```

Comment:
these two functions are a little oblique to me (probably because I don't know React and TS very well). Maybe add some comments describing what they do and why?

Reply:
Added some comments for those!